odoo/addons/base/tests/test_image.py

```diff
@@ -177,6 +176,20 @@ def test_13_image_process_quality(self):
         res = tools.image_process(self.img_1920x1080_jpeg)
         self.assertLessEqual(len(res), len(self.img_1920x1080_jpeg))
 
+        # CASE: JPEG optimize + bigger size => original
+        image = Image.new('RGB', (1920, 1080), color=self.bg_color)
+        offset = (image.size[0] - image.size[1]) / 2
+        draw = ImageDraw.Draw(image)
+        draw.ellipse(xy=[
+            (offset, 0),
+            (image.size[0] - offset, image.size[1])
+        ], fill=self.fill_color, outline=(240, 25, 40), width=10)
```

Comment:
I don't understand what is this draw stuff and why is it needed ?

Reply:
To compare compressed sizes, I want the image to have an actual content, and preferably not a rectangular one - so, a single ellipse shape sounded good enough for the test purpose - and it avoids relying on an external data file.

Reply:
Strange that others image tests don't do that.. Are you sure there is no other/simpler way to have a non void image ?